src/lib.rs

```diff
@@ -187,6 +187,25 @@ fn concat<T>(mut v1: Vec<T>, mut v2: Vec<T>) -> Vec<T> {
     v1
 }
 
+fn check_for_default(triples: &mut Vec<(TokenStream, Ident, TokenTree)>) {
+    let mut default_position: Option<usize> = None;
+    for (attributes, variant_name, variant_value) in triples.into_iter() {
+        if attributes.to_string().contains("default") {
+            if default_position.is_some() {
+                // error!("Multiple variants marked as default");
+            }
+            default_position = Some(variant_value.to_string().parse::<usize>().unwrap());
```

Comment:
You could do a return here instead. And if you do, you won't need to do `if !default_position.is_some()` below.